usaspending_api/etl/management/commands/load_query_to_delta.py

```diff
@@ -328,6 +343,12 @@ def add_arguments(self, parser):
             help="An alternate delta table name for the created table, overriding the TABLE_SPEC destination_table "
             "name",
         )
+        parser.add_argument(
```

Comment:
What happens with CDF if it is enabled on a table but then someone runs this command and does a full reload of the table? Is this a case we need to protect against?